### PR TITLE
Coding convention applied split 2-5-2 (of 2-14-2)

### PR DESCRIPTION
**Pull request by Quix0r**
This PR is the next of **14** upcoming splits of the first series (yes, a lot has been changed) which started with #3254 (origin: #3112 ). It includes coding-convention stuff like added spaces and curly braces. Also some `dbm::is_result()` may have been included.

The commit has been split out from a larger commit containing 192 (!) changed files.

More similar changes: (in order they are created)

- [x] part 2 #3254 
- [x] part 2-1-2 #3287
- [x] part 2-2-2 #3369
- [x] part 2-3-2 #3375
- [x] part 2-4-2 #3381
- [ ] part 2-5-2 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split2-5-2 (this)
- [ ] part 2-6-2 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split2-6-2
- [ ] part 2-7-2 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split2-7-2
- [ ] part 2-8-2 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split2-8-2
- [ ] part 2-9-2 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split2-9-2
- [ ] part 2-10-2 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split2-10-2
- [ ] part 2-11-2 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split2-11-2
- [ ] part 2-12-2 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split2-12-2
- [ ] part 2-13-2 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split2-13-2
- [ ] part 2-14-2 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split2-14-2
- [ ] part 2-2 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split2-2 (**final!**)
- [ ] part 3 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split3
- [ ] part 4 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split4
- [ ] part 5 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split5
- [ ] part 6 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split6
- [ ] / : https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-root
- [ ] utils/ : https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-utils
- [ ] mod/ https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-mod
- [ ] include 1: https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-include-if
- [ ] include 2: https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-include-foreach
- [ ] others: https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-others
- [ ] else-if: https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-else-if

Unrelated but comes next:
- [ ] chmod https://github.com/Quix0r/friendica/tree/fixes/chmod

### CODE REVIEW

**Comment by Hypolite** [comment]
Did you run `php util/typo.php` ?

**Comment by Quix0r** [comment]
Ah, `dba_pdo.php` is broken. I don't use it on my server. I have not added it to `post-commit` hook as this makes commits very long.

**Comment by Hypolite** [comment]
I doesn't make sense as a `post-commit` hook but it definitely is an important tool to detect early mistakes when you're ready to submit your PR.

**Comment by Quix0r** [comment]
Sure. :-)

**Comment by Hypolite** [comment]
I'll pull it tonight just to be sure it doesn't break anything obvious and I'll probably merge it in the same movement.

File: include/queue.php [hunk 1]
```diff
@@ -29,10 +29,29 @@ function queue_run(&$argv, &$argc){
 		// Handling the pubsubhubbub requests
 		proc_run(PRIORITY_HIGH,'include/pubsubpublish.php');
 
+		$interval = ((get_config('system','delivery_interval') === false) ? 2 : intval(get_config('system','delivery_interval')));
```
Comment (rabuzarus) [inline include/queue.php]: The whole part was removed because we now only use the worker

Comment (Quix0r) [inline include/queue.php]: I then have to remove it again, okay. Thank you for finding it. Have to go now.

File: include/diaspora.php [hunk 2]
```diff
@@ -3777,8 +3777,10 @@ public static function store_like_signature($contact, $post_id) {
 		$message = self::construct_like($r[0], $contact);
 		$message["author_signature"] = self::signature($contact, $message);
 
-		// We now store the signature more flexible to dynamically support new fields.
-		// This will break Diaspora compatibility with Friendica versions prior to 3.5.
+		/*
+		 * Now store the signature more flexible to dynamically support new fields.
```
Comment (annando) [inline include/diaspora.php]: Why have you changed the sentence?

Comment (Quix0r) [inline include/diaspora.php]: I like them to be more neutral and descriptive.

**Pull request merged**